contracts/common/GetActionSummary.sol

```diff
@@ -147,17 +144,16 @@ contract GetActionSummary is ExtractCallData, GetActionDomainSkillId {
         totalSummary.domainSkillId = actionSummary.domainSkillId;
```

Comment:
I believe this block could be re-written as:

```
require(
  totalSummary.domainSkillId == 0 || totalSummary.domainSkillId == actionSummary.domainSkillId,
  "colony-action-summary-inconsistent-domain-skill-id"
)

totalSummary.domainSkillId = actionSummary.domainSkillId;
```

I might even move it towards the bottom next to the `requiredPermissions` check.

Reply:
Moving it to the required permissions check is 100% better, totally agree with that. I'm always a little hesitant about refactors like the one you're proposing here, because I remain convinced that the `if` syntax is more understandable (mostly because, when understanding a block like this, one naturally parses it in to English using the word "if").

That said, I have done it :smile:

Reply:
I've also refactored another similar section you didn't highlight for consistency.